packages/kbn-unified-data-table/src/components/data_table_columns.tsx

```diff
@@ -267,9 +267,18 @@ export function getEuiGridColumns({
   );
 }
 
-export function getVisibleColumns(columns: string[], dataView: DataView, showTimeCol: boolean) {
+export function getVisibleColumns(
+  columns: string[],
+  dataView: DataView,
+  showTimeCol: boolean,
+  isPlainRecord: boolean = false
+) {
   const timeFieldName = dataView.timeFieldName;
 
+  if (isPlainRecord) {
```

Comment:
This hides the @timestamp column all the time. We don't want that. We just want to hide it when the user is dropping the field